src/psyclone/transformations.py

```diff
@@ -1519,24 +1525,27 @@ class OMPSingleTrans(ParallelRegionTrans):
     >>> from psyclone.parse.algorithm import parse
     >>> from psyclone.psyGen import PSyFactory
     >>> api = "gocean1.0"
-    >>> filename = "nemolite2d_alg.f90"
-    >>> ast, invokeInfo = parse(filename, api=api, invoke_name="invoke")
+    >>> ast, invokeInfo = parse(SOURCE_FILE, api=api)
     >>> psy = PSyFactory(api).create(invokeInfo)
     >>>
     >>> from psyclone.transformations import OMPParallelTrans, OMPSingleTrans
     >>> singletrans = OMPSingleTrans()
     >>> paralleltrans = OMPParallelTrans()
     >>>
     >>> schedule = psy.invokes.get('invoke_0').schedule
-    >>> schedule.view()
+    >>> if DEBUG:
+    ...    schedule.view()
+    >>> if DEBUG:
```

Comment:
So good you replaced it twice? ;-)

Reply:
Yes, that's just the best, so can't hurt to have it twice :)